spec/api-content-tracing-spec.js

```diff
@@ -115,6 +111,21 @@ describe('contentTracing', () => {
   describe('stopRecording', function () {
     this.timeout(5e3)
 
+    it('does not crash on empty string', () => {
+      const options = {
+        categoryFilter: '*',
+        traceOptions: 'record-until-full,enable-sampling'
+      }
+
+      expect(() => {
+        contentTracing.startRecording(options).then(() => {
+          setTimeout(() => {
+            contentTracing.stopRecording('').then(path => console.log(path))
+          }, 5000)
```

Comment:
is there any particular need for this test to take 5 seconds? Can we just immediately call `stopRecording` with no timeout?

Can we also check that `path` exists & is a file?

Reply:
my understanding was that it is to emulate some tracing delay, there should be no reason to not do it immediately if needed.

Reply:
done!